autoware_launch/launch/e2e_simulator.launch.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<launch>
+  <!-- Essential parameters -->
+  <arg name="map_path" description="point cloud and lanelet2 map directory path"/>
+  <arg name="vehicle_model" description="vehicle model name"/>
+  <arg name="sensor_model" description="sensor model name"/>
+
+  <!-- Optional parameters -->
+  <!-- Modules to be launched -->
+  <arg name="vehicle" default="true" description="launch vehicle"/>
+  <arg name="system" default="true" description="launch system"/>
+  <arg name="map" default="true" description="launch map"/>
+  <arg name="sensing" default="true" description="launch sensing"/>
+  <arg name="localization" default="true" description="launch localization"/>
+  <arg name="perception" default="true" description="launch perception"/>
+  <arg name="planning" default="true" description="launch planning"/>
+  <arg name="control" default="true" description="launch control"/>
+  <!-- Pointcloud container -->
+  <arg name="use_pointcloud_container" default="true" description="launch pointcloud container"/>
+  <arg name="pointcloud_container_name" default="pointcloud_container"/>
+  <!-- Map -->
+  <arg name="lanelet2_map_file" default="lanelet2_map.osm" description="lanelet2 map file name"/>
+  <arg name="pointcloud_map_file" default="pointcloud_map.pcd" description="pointcloud map file name"/>
+  <!-- Vehicle -->
+  <arg name="launch_vehicle_interface" default="false"/>
+  <!-- Tools -->
+  <arg name="rviz" default="true" description="launch rviz"/>
+  <arg name="rviz_config" default="$(find-pkg-share autoware_launch)/rviz/autoware.rviz" description="rviz config"/>
+
+  <group scoped="false">
+    <include file="$(find-pkg-share autoware_launch)/launch/autoware.launch.xml">
+      <!-- Common -->
+      <arg name="map_path" value="$(var map_path)"/>
+      <arg name="vehicle_model" value="$(var vehicle_model)"/>
+      <arg name="sensor_model" value="$(var sensor_model)"/>
+      <!-- Modules to be launched -->
+      <arg name="launch_vehicle" value="$(var vehicle)"/>
+      <arg name="launch_map" value="$(var map)"/>
+      <arg name="launch_sensing" value="$(var sensing)"/>
+      <arg name="launch_localization" value="$(var localization)"/>
+      <arg name="launch_perception" value="$(var perception)"/>
+      <arg name="launch_planning" value="$(var planning)"/>
+      <arg name="launch_control" value="$(var control)"/>
+      <!-- Global parameters -->
+      <arg name="use_sim_time" value="true"/>
+      <!-- Pointcloud container -->
+      <arg name="use_pointcloud_container" value="$(var use_pointcloud_container)"/>
+      <arg name="pointcloud_container_name" value="$(var pointcloud_container_name)"/>
```

Comment:
I think it is better to remove to reduce the difference between autoware.launch and e2e.launch

Reply:
Thank you. I removed them by 3d8af47df1b112340056e0a2d7c25dc4b8b3ebd7.